texar/torch/modules/pretrained/pretrained_base.py

```diff
@@ -181,8 +181,7 @@ def download_checkpoint(cls, pretrained_model_name: str,
         Args:
             pretrained_model_name (str): Name of the model checkpoint.
             cache_dir (str, optional): Path to the cache directory. If `None`,
-                uses the default directory given by
-                :meth:`~default_download_dir`.
+                uses the default directory (user's home directory).
```

Comment:
Perhaps a better default directory is the current working directory instead of home?

Reply:
This was changed in issue #146 because we now use `pip install` without `-e`. `pip` would scan all files and having very large files under the project directory would make it unbearably slow (took a few minutes; I thought it froze at first). Other libraries (e.g. NLTK) also download stuff to a directory under home so I think it's okay. Plus, Texar-TF & Texar-PyTorch can share downloaded resources.

Reply:
I see. Make sense